var/spack/repos/builtin/packages/mpidiff/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Mpidiff(CMakePackage):
```

Comment:
Is there a convention in spack that prevents this class from being named MPIDiff?

Reply:
All package names are lower-case, with hyphens as needed.

The associated class name for `foo-bar` is `FooBar`. So a class named `MPIDiff` would be `m-p-i-diff` on the command line.